src/extension/android/androidPlatform.ts

```diff
@@ -101,7 +102,7 @@ export class AndroidPlatform extends GeneralMobilePlatform {
                         .finally(() => {
                             return this.initializeTargetDevicesAndPackageName();
                         }).then(() => [this.debugTarget], reason => {
-                            if (reason.message === AndroidPlatform.MULTIPLE_DEVICES_ERROR && this.devices.length > 1 && this.debugTarget) {
+                            if (reason.message === ErrorHelper.getInternalError(InternalErrorCode.AndroidMoreThanOneDeviceOrEmulator).message && this.devices.length > 1 && this.debugTarget) {
```

Comment:
Would this condition work correctly after localization of error?

Reply:
This condition is deprecated, but I replaced it because of our androidPlatform class tests.